### PR TITLE
new changes

### PR DESCRIPTION
**Pull request by snigdhaalekhya**

### CODE REVIEW

File: app/controllers/main_controller.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ class MainController < ApplicationController
 
     def index
         @carts= Cart.of_user(current_user)
+        @orders= Order.of_user(current_user)
         if params[:search].blank?
```
Comment (vaishalisrinivasan) [inline app/controllers/main_controller.rb:8]: why if is used with no operation inside?

Comment (vaishalisrinivasan) [reply app/controllers/main_controller.rb:8]: please remove the unused methods

Comment (vaishalisrinivasan) [inline app/controllers/main_controller.rb:8]: WHY 'IF' IS USED WITHOUT ANY OPERATIONS?

Comment (vaishalisrinivasan) [reply app/controllers/main_controller.rb:8]: please remove the unused methods

File: app/controllers/menus_controller.rb [hunk 2]
```diff
@@ -54,7 +54,7 @@ def edit
     def update
       id=params[:id]
```
Comment (vaishalisrinivasan) [inline app/controllers/menus_controller.rb:55]: please don't assign variable if not necessary

**Comment by vaishalisrinivasan** [comment]
Please remove the unused methods and lines

File: app/controllers/menus_controller.rb [hunk 3]
```diff
@@ -16,12 +16,12 @@ def create
             menu_category: params[:menu_category],
             menu_cost: params[:menu_cost],
             menu_description: params[:menu_description],
+            menu_image: params[:menu_image]
```
Comment (vaishalisrinivasan) [inline app/controllers/menus_controller.rb:19]:
```
def index   
      if Owner.find_by(email: current_owner.email)
        @owner_worker= current_owner.email
      elsif Worker.find_by(email: current_owner.email)
        @owner_worker= current_owner.email
      end  
    end
```


try using define_method

Comment (vaishalisrinivasan) [reply app/controllers/menus_controller.rb:19]: can we do the payload construction in a separate method within the private method

```
menu=Menu.new(
            menu_name: params[:menu_name],
            menu_category: params[:menu_category],
            menu_cost: params[:menu_cost],
            menu_description: params[:menu_description],
            menu_image: params[:menu_image]
```

File: app/controllers/orders_controller.rb [hunk 4]
```diff
@@ -2,8 +2,8 @@ class OrdersController < ApplicationController
     
     def index
          @carts=Cart.of_user(current_user) 
-         @orders= Order.of_user(current_user)
-         @orders=@orders.sort_by(&:updated_at).reverse
+         @orders_user= Order.of_user(current_user)
+         @orders=@orders_user.sort_by(&:updated_at).reverse
```
Comment (vaishalisrinivasan) [inline app/controllers/orders_controller.rb:6]: what exactly are you trying to do here?

Comment (snigdhaalekhya) [reply app/controllers/orders_controller.rb:6]: I am using them in index.html.erb file (the variables)

**Pull request merged**